### PR TITLE
Fixed pichia error by adding multiline genbank feature locations

### LINKED ISSUE #124: Pichia chromosome parsing fails
**Describe the bug**
Parsing a Pichia chromosome genbank file (https://www.ncbi.nlm.nih.gov/nuccore/FR839628.1) fails at a multiline location. In particular:

```
     mRNA            join(<459260..459456,459556..459637,459685..459739,
                     459810..>460126)
```

**To Reproduce**
Download linked genbank file and try to parse it. You will get a panic. 

**Expected behavior**
A successful location parse

### PR DESCRIPTION
**Pull request by Koeng101**
Closes #124 

I fixed the error by allowing multiline genbank feature locations. I don't know what the heck `join(<459260..459456,459556..459637,459685..459739,459810..>460126)` means... but the parsing works, at least.

### CODE REVIEW

**Comment by Koeng101** [comment]
Sorry code climate, parsing genbank files is complex.

**Comment by Koeng101** [comment]
Wait, there could be an error later in the parser at 

```
 // loop through any potential continuing lines of qualifiers. Break if not.
                        for {
                                // keeping out of normal if else pattern because of phantom brackets that are hard to trace.
                                if !quickQualifierSubLineCheck(line) {
                                        break
                                }
                                //append to current qualifier
                                // qualifier += strings.TrimSpace(line)
                                if qualifierKey != "/translation" {
                                        qualifier += " " + strings.TrimSpace(line)
                                } else {
                                        qualifier += strings.TrimSpace(line)
                                }

                                // nextline
                                lineIndex++
                                line = lines[lineIndex]
                        }
```
we'd want to reset to after wherever the multiline parser dropped off.

File: io_test.go [hunk 1]
```diff
@@ -169,6 +169,16 @@ func TestGbkIO(t *testing.T) {
...
 		t.Errorf("Parsing the output of BuildGbk() does not produce the same output as parsing the original file read with ReadGbk(). Got this diff:\n%s", diff)
 	}
+
+	// Test multiline Genbank features
+	pichia := ReadGbk("data/pichia_chr1_head.gb")
+	var multilineOutput string
+	for _, feature := range pichia.Features {
+		multilineOutput = feature.GbkLocationString
+	}
+	if multilineOutput != "join(<459260..459456,459556..459637,459685..459739,459810..>460126)" {
```
Comment (TimothyStiles) [inline io_test.go:179]: Do you know if this eventually parses in json @Koeng101?

Comment (Koeng101) [reply io_test.go:179]: No. How would I learn if it does?

**Comment by Koeng101** [comment]
```json
"gbk_location_string": "join(\u003c459260..459456,459556..459637,459685..459739,459810..\u003e460126)",
   "sequence": "",
   "sequence_location": {
    "start": 0,
    "end": 0,
    "complement": false,
    "join": true,
    "five_prime_partial": true,
    "three_prime_partial": true,
    "sub_locations": [
     {
      "start": 459259,
      "end": 459456,
      "complement": false,
      "join": false,
      "five_prime_partial": true,
      "three_prime_partial": false,
      "sub_locations": null
     },
     {
      "start": 459555,
      "end": 459637,
      "complement": false,
      "join": false,
      "five_prime_partial": false,
      "three_prime_partial": false,
      "sub_locations": null
     },
     {
      "start": 459684,
      "end": 459739,
      "complement": false,
      "join": false,
      "five_prime_partial": false,
      "three_prime_partial": false,
      "sub_locations": null
     },
     {
      "start": 459809,
      "end": 460126,
      "complement": false,
      "join": false,
      "five_prime_partial": false,
      "three_prime_partial": true,
"sub_locations": null
     }
    ]
   },
```

It does the right parsing.

**Pull request merged**